libcudacxx/test/libcudacxx/std/linalg/layouts/transposed.pass.cpp

```diff
@@ -0,0 +1,137 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++11, c++14
+// UNSUPPORTED: msvc && c++17
+
+#include "cuda/std/__linalg/transposed.h"
```

Comment:
This should include the proper public header and not a subheader
```suggestion
#include <cuda/std/linalg>
```